packages/strapi-design-system/src/ModalLayout/__tests__/ModalLayout.spec.js

```diff
@@ -164,7 +164,8 @@ describe('ModalLayout', () => {
         color: #666687;
       }
 
-      .c15[aria-disabled='true'] svg > g,.c15[aria-disabled='true'] svg path {
+      .c15[aria-disabled='true'] svg > g,
```

Comment:
ℹ️ Not to sure why I had to update the snapshot for this formatting diff after rebasing 🤔

Reply:
did you yarn install after merging main?

Reply:
yeah, I had a merge conflict with yarn.lock, so I ran yarn

Reply:
but something feels off, because the e2e test is failing now

Reply:
This looks like a formatting change to me so I don't think you need to worry too much.